scripts/onebranch/post-build.ps1

```diff
@@ -11,59 +11,27 @@ Set-Location $scriptPath\..\..
 
 $OneBranchArch = $env:ONEBRANCH_ARCH
 $OneBranchConfig = $env:ONEBRANCH_CONFIG
+$SignedBinFolder = ".\build\bin\$($OneBranchArch)_$($OneBranchConfig)"
+$DestinationFolder = ".\build\bin\$($OneBranchArch)_$($OneBranchConfig)"
```

Comment:
Any reason you didn't just assign `$DestinationFolder = $SignedBinFolder`? This is just for my understanding, not suggesting this should be changed.

Reply:
let me add some comments and use a single variable.